services/node_stake_update.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+import "basic_types.proto";
+import "timestamp.proto";
+
+/**
+ * Updates the staking info at the end of staking period to indicate new staking period has started.
+ */
+message NodeStakeUpdateTransactionBody {
```

Comment:
Does this need to go in any other files like `HederaFunctionality` enum, a new `NodeService`, etc?

Reply:
Since we wont be implementing any `getNodeInfo` query or any transition logic on services side and using the txn body only for record stream , it is not needed to add `NodeService`. Will mirror node use `HederaFunctionality` enum?

Reply:
We use it in tests to verify our transaction types is up to date. Think we should update it.

Reply:
I see . Updated. Thanks